wagtail/admin/templates/wagtailadmin/shared/action_switch.html

```diff
@@ -0,0 +1,22 @@
+{% load wagtailadmin_tags %}
+{% comment %}
+    This is a switch (checkbox) input that is hooked into the ActionController
+    to allow for a POST request to be made when the switch is toggled.
+
+    Variables this template accepts:
+
+    data_url - A URL for ActionController to use
+    checked - Whether the switch is checked (active) or not
+    label_text - The text that shows up beside the switch
+    sr_only_label - Make the label invisible for all but screen reader users
+    classname - Custom CSS classes for styling
```

Comment:
Yeah I assumed there might be use cases where these would be necessary, but for now we don't even have other instances of this small component, so it might've been too early to add these!